### ISSUE TITLE
Attribute values throwing warning

### ISSUE THREAD

**Issue by PalleZingmark** [rossPatton/stylint#212]
Using `v1.2.3`:

```css
div:not([style="display: block;"]) {
  display: none;
}
```
`-> Warning: avoid one liners. put properties on their own line`

**Comment by rossPatton** [comment]
@PalleZingmark can i get more information about your config? i am not seeing this locally

**Comment by PalleZingmark** [comment]
@rossPatton Certainly, here´s how it's setup:


`gulp v3.9.0`
`gulp-shell v0.5.0`
`stylint v1.2.3`

**gulpfile.js**
``` javascript
gulp.task("stylint", shell.task([
  "stylint ./app/assets/stylus/ -c .stylintrc"
]));
```

**.stylintrc**
```json
{
  "blocks": false,
  "brackets": "always",
  "colons": "always",
  "colors": "always",
  "commaSpace": "always",
  "commentSpace": "always",
  "cssLiteral": "never",
  "depthLimit": 4,
  "duplicates": true,
  "efficient": "always",
  "extendPref": false,
  "globalDupe": false,
  "indentPref": 2,
  "leadingZero": "never",
  "maxErrors": false,
  "maxWarnings": false,
  "mixed": true,
  "namingConvention": "BEM",
  "namingConventionStrict": true,
  "none": "never",
  "noImportant": true,
  "parenSpace": "never",
  "placeholders": "always",
  "prefixVarsWithDollar": "always",
  "quotePref": "double",
  "semicolons": "always",
  "sortOrder": false,
  "stackedProperties": "never",
  "trailingWhitespace": "never",
  "universal": false,
  "valid": true,
  "zeroUnits": "never",
  "zIndexNormalize": false
}
```

**Comment by rossPatton** [comment]
thanks, i'll take another look tonight

**Comment by rossPatton** [comment]
still can't replicate this. 

i've tested the following ways with your config:
- locally via cli 
- locally via gulp
- locally used directly 
- using 1.2.3 cli
- using 1.2.3 via gulp
- using 1.2.3 directly

sorry to not be more help.

closing for now (if anyone else has this issue I will re-open).

**Comment by PalleZingmark** [comment]
@rossPatton This is so weird. I've tested this issue now on 5 different setups and they all generate the same warning. `-> Warning: avoid one liners. put properties on their own line`

Here is the exact code I've tested on: https://www.dropbox.com/sh/5f531dyxiosb66e/AAC3YDwbH6R9ImwPvhUqy9dFa?dl=0 

1. `cd stylint-test`
2. `npm install stylint` 
3. `stylint test.styl`

![image](https://cloud.githubusercontent.com/assets/367036/10752247/d1fcd886-7c87-11e5-8851-6d37c0d925ce.png)

**Comment by carsotho** [comment]
I also get this bug when running @PalleZingmark test case with stylint 1.2.5, with Node 0.12.7 on Mac OSX 10.10.5

**Comment by PalleZingmark** [comment]
Found another interesting issue regarding `Warning: avoid one liners. put properties on their own line` today, not sure it's really related to this issue, but I'll post here first instead of opening a new issue.

I used the same setup, but with this code: 
(note: there is a white-space after _display: block;_ as better seen in the screenshot) 
```css
.foo {
  display: block; 
}
```
> Warning: avoid one liners. put properties on their own line, Line: 2: display: block;
> Warning: trailing whitespace, Line: 2: display: block;

And when I removed the whitespace, I got no warnings at all.

![image](https://cloud.githubusercontent.com/assets/367036/10771857/6548bcb6-7cf3-11e5-95da-71283d57f1fa.png)

**Comment by rossPatton** [comment]
that is very weird. hmmm

**Comment by PalleZingmark** [comment]
If I look at Line 4 in [stylint/src/checks/stackedProperties.js](https://github.com/rossPatton/stylint/blob/1f3b25b9be6e993d8739fc7c087c428f582c5819/src/checks/stackedProperties.js#L3), does that regexp take a trailing whitespace into the calculation?
>// if semicolons on line, but not ending the line, prolly a one-liner
>`var semiTest = /;+(?!$)/gm`

**Comment by rossPatton** [comment]
prolly not. i trim my trailing whitespaces by default (via sublime settings) so most likely i missed that case

**Comment by rossPatton** [comment]
so, this wasn't in the latest release (1.3.0) but i hope to get to it for 1.3.1

**Comment by rossPatton** [comment]
this should be fixed in 1.3.2
